src/pages/docs/projects/export-import/index.mdx

```diff
@@ -207,3 +246,7 @@ You can use the [Octopus REST API](/docs/octopus-rest-api) to export or import O
 To find out more take a look at our examples:
 - [Export projects](/docs/octopus-rest-api/examples/projects/export-projects)
 - [Import projects](/docs/octopus-rest-api/examples/projects/import-projects)
+
+## Older versions
+
+- Prior to version **2025.2.5601**, version-controlled projects were not supported by the **Export/Import Projects** feature.
```

Comment:
I am no expert at English, it feels like this could have the logic reversed:

```suggestion
- Importing and Exporting of version-controlled projects is only supported from version **2025.2.5601**
```

Reply:
I agree that it feels backward when you read it, but it's the logical process you go through though. And we use that order elsewhere in these docs.

Reply:
If it is used elsewhere then leave it as is